pkg/sql/logictest/testdata/logic_test/pgcrypto_builtins

```diff
@@ -147,9 +147,125 @@ $2a$04$ 29
 $2a$31$ 29
 
 # invalid min rounds
-statement error pgcode 22023 incorrect number of rounds
+statement error pgcode 22023 invalid number of salt rounds
 SELECT gen_salt('bf', 3)
 
 # invalid max rounds
-statement error pgcode 22023 incorrect number of rounds
+statement error pgcode 22023 invalid number of salt rounds
 SELECT gen_salt('bf', 32)
+
+subtest crypt_invalid_algo
```

Comment:
i know we don't do this much, but it's good practice to put `subtest end` after your subtest in case more tests get added